api-reference/50 Common/Object Structures/format/useCurrencyAccountingStyle.md

```diff
@@ -5,7 +5,12 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to apply the accounting style to formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The accounting style adds parentheses to negative numbers instead of the minus sign. To disable the accounting style in the component and display formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type) with the minus sign, set this property to **false**. This property overrides the [global config](/Documentation/ApiReference/Common/Object_Structures/globalConfig/#defaultUseCurrencyAccountingStyle).
```

Comment:
1. Do I get this right that if I leave this property unspecified, the resulting style will be determined by the global accounting style flag? If this is correct, I believe we should explain it in a more explicit manner.
2. The global **config**. "Config" does not sound as an ordinary noun, this is a shorthand of some sort from the sound of it. Is it a global configuration file? Or just a set of settings? Is "global config" a proper noun? If you call it "global config" everywhere across your documentation and customers understand it, then I have no objections. I just had to clarify this.

Reply:
I tried to explain it better and also added an image in both files to illustrate the link between these properties.

Also I removed the word "config".